charts/civil-ga-ccd/values.preview.template.yaml

```diff
@@ -72,7 +72,7 @@ civil-service:
     releaseNameOverride: ${SERVICE_NAME}-civil-service
     image: 'hmctspublic.azurecr.io/civil/service:latest'
     imagePullPolicy: Always
-    ingressHost: civil-service-${SERVICE_FQDN}
+    ingressHost: civil-service-${SERVICE_FQDN}.preview.platform.hmcts.net
```

Comment:
this is wrong please verify . .it should be service name . similar to other changes